apps/dapp-console-api/docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+version: '3.8'
+
+services:
+  redis:
+    image: 'redis:latest'
+    healthcheck:
+      test: ['CMD', 'redis-cli', 'ping']
+      interval: 5s
+      timeout: 10s
+      retries: 3
+      start_period: 2s
+    volumes:
+      - redis_data:/data/redis
+    ports:
+      - 6380:6379
+
+  api:
+    build:
+      context: ../../
+      dockerfile: Dockerfile
+      target: dapp-console-api
+    env_file: .env
+    environment:
+      - REDIS_URL=redis://redis:6379
+    healthcheck:
+      test: curl http://0.0.0.0:7300/api/v0/healthz
+    volumes:
+      - ../../certs/extra-ca-certificates.crt:/usr/local/share/ca-certificates/extra-ca-certificates.crt
+    ports:
+      - 7310:7300
```

Comment:
nit for later: wdyt about making this port something other than 7310 so it doesn't clash with paymaster-proxy

Reply:
I can update that now!